docs/eventing/channel-based-broker.md

```diff
@@ -53,6 +58,18 @@ data:
       replicationFactor: 1
 ```
 
+**NOTE:** This example specifies two extra parameters that are specific to Kakfa Channels; `numPartitions` and `replicationFactor`.
+
+## Configuring the MT Broker
+
+After you have configured the ConfigMap for each Channel provider type, you can configure the MT Broker.
+You can configure which Channels are used as a cluster level default, by namespace or for a specific Broker.
+The Channels used are configured by a `config-br-defaults` ConfigMap in the `knative-eventing` namespace.
+
+The following example ConfigMap uses a Kafka channel for all Brokers, except for the `test-broker-6` namespace,  which uses InMemoryChannel.
```

Comment:
maybe change test-broker-6 to something else, like imc-namespace or something like that. this was just one of the test namespaces I was using while writing the docs :) 
And if you do that, it has ripple effects after this point.